server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -132,9 +132,23 @@
    * avoid some tsfileResource is changed (e.g., from unsealed to sealed) when a query is executed.
    */
   private final ReadWriteLock closeQueryLock = new ReentrantReadWriteLock();
-  private final HashMap<Long, TsFileProcessor> workSequenceTsFileProcessor = new HashMap<>();
+  /**
+   * time range id for divide storage group -> tsFileProcessor for this storage group
```

Comment:
time range id for divide storage group -> time range id in the storage group
tsFileProcessor for this storage group -> tsFileProcessor for this time range

Reply:
fixed~